member-event-guidelines.md

```diff
@@ -157,6 +162,8 @@ benefits.
     products. Events may also receive assorted swag from MLH Season
     sponsors such as pens and t-shirts.
 
+    - **Asia-Pacific:** Select swag may not be available to Member Events in Asia-Pacific.
```

Comment:
Can we have a discussion on this?

Reply:
For now we should leave this in. We can't guarantee that every single thing we provide swag wise in North America and Europe is available in APAC. Most of it is, but this phrasing appropriately sets expectations.